source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -102,6 +102,10 @@ the feature cannot determine what to do and the operation will fail.
 
 * The cluster's management network stays on the original interface which is excluded from the operation. 
 
+* QoS, DNS and custom_properties are currently unsupported.
```

Comment:
'currently unsupported' suggest there is a plan somehow somewhere to make it supported - if there are already bz's to support this please add them so that whoever is interested will have a way to track their progress, or if there aren't I think it's perfectly fine to write they are simply unsupported - if in the future that will change then obviously this document will change as well.

Reply:
Since this is in limitation, I just listen the features that are unsupported. I hope it is ok, wanted it to be black and white.